<h4>Assignment 2 Problem 1: Washing Machine</h4>

As the washtub of a front-loading industrial washing
machine turns around its axis, the laundry and wash
water are pushed to one side so that the centroid of the
washtub's contents lies off-center, a distance $a=1.0\,m$
from the central axis. The washing machine rests on
wheels, and is spring-mounted to a rigid wall support (as
illustrated in the accompanying diagram). Assuming
that the motion is resisted by padding on either side of
the machine, the horizontal displacement $x$ along the
floor as a function of time t is governed by the
differential equation (neglecting friction):
$$
M\frac{d^{2}x}{dt^{2}}+c\frac{dx}{dt}+kx=ma\omega^{2}\cos(\omega{}t)
$$
where $m=36\,kg$ is the mass of the washtub's
contents, $M=101\,kg$ is the mass of the entire
washing machine (including the washtub's
contents), $k=793\,N/m$ is the spring constant of
its elastic connection to the support and
$c=176\,kg/s$ is the damping constant.

<img src="./diagrams/nelkyqye1.png" width="40%" height="40%">

* Determine the complementary part of the solution for x.
* Find the particular part of the solution for $x$ by the method of undetermined
coefficients. Determine values for the coefficient $A$ of the cosine term and $B$ of the
sine term.
* Determine the maximum amplitude $x_{max}$ of the oscillations over time. Use Maple or
Excel to plot the amplitude $x_{max}$ as a function of the angular frequency $\omega$ from $0$ to $5$
radians per second. If you use Excel, plot the angular frequency $\omega$ using an
increment of $0.1$ radians per second.
* Determine the angular frequency $\omega$ at which the amplitude $x_{max}$ reaches its resonant
value. What is the corresponding amplitude at this angular frequency? Give your
answers to three significant figures and include the units in your answer.

The differential equation is
$$
101x''+176x'+793x=36\omega^{2}\cos(\omega{}t)
$$

In [1]:
using Polynomials

In [4]:
p2=Polynomial([793,176,101])

Polynomial(793 + 176*x + 101*x^2)

In [5]:
roots(p2)

2-element Vector{ComplexF64}:
 -0.8712871287128712 - 2.6631454875492877im
 -0.8712871287128712 + 2.6631454875492877im

In [31]:
oms=imag(roots(p2)[2,1])

2.6631454875492877

Therefore,
$$
x_{c}(t)=e^{-0.87t}\left(c_{1}\cos(2.66t)+c_{2}\sin(2.66t)\right)
$$

In [6]:
using SymPy

In [7]:
@vars w

(w,)

In [8]:
A1=[-101*w^2+793 176*w
-176*w -101*w^2+793]

2×2 Matrix{Sym}:
 793 - 101*w^2          176⋅w
        -176⋅w  793 - 101*w^2

In [9]:
b1=[36*w^2
0]

2-element Vector{Sym}:
 36*w^2
      0

In [13]:
A2=inv(A1)*b1

2-element Vector{Sym}:
 -36*w^2*(101*w^2 - 793)/(10201*w^4 - 129210*w^2 + 628849)
                6336*w^3/(10201*w^4 - 129210*w^2 + 628849)

$$
x_{p}(t)=- \frac{36 {\omega}^{2} \left(101 {\omega}^{2} - 793\right)}{10201 {\omega}^{4} - 129210 {\omega}^{2} + 628849}\cos(\omega{}t)+\frac{6336 {\omega}^{3}}{10201 {\omega}^{4} - 129210 {\omega}^{2} + 628849}\sin(\omega{}t)
$$

In [11]:
p3=Polynomial([628849,0,129210,0,10201])

Polynomial(628849 + 129210*x^2 + 10201*x^4)

In [12]:
roots(p3)

4-element Vector{ComplexF64}:
 -0.8712871287128717 - 2.6631454875492886im
 -0.8712871287128717 + 2.6631454875492886im
  0.8712871287128698 - 2.663145487549289im
  0.8712871287128698 + 2.663145487549289im

In [23]:
C=sqrt(A2[1,1]^2+A2[2,1]^2) # this is the amplitude of the steady periodic function in terms of omega

        _____________________________________________________________________
       ╱                                                              2      
      ╱                      6                        4 ⎛     2      ⎞       
     ╱             40144896⋅w                   1296⋅w ⋅⎝101⋅w  - 793⎠       
    ╱    ──────────────────────────────── + ──────────────────────────────── 
   ╱                                    2                                  2 
  ╱      ⎛       4           2         ⎞    ⎛       4           2         ⎞  
╲╱       ⎝10201⋅w  - 129210⋅w  + 628849⎠    ⎝10201⋅w  - 129210⋅w  + 628849⎠  

<img src="./diagrams/nelkyqye-desmos.png" width="40%" height="40%">

In [26]:
C1(om)=sqrt(((40144896*om^6)/((10201*om^4-129210*om^2+628849)^2))+((1296*om^4*((101*om^2-793)^2))/((10201*om^4-129210*om^2+628849)^2)))

C1 (generic function with 1 method)

In [27]:
C1(3.12)

0.6030408775735087

The angular frequency $\omega$ at which the amplitude $x_{max}$ reaches its resonant
value is $\omega=3.12$. The corresponding amplitude at this angular frequency is approximately $0.603$.